src/App.js

```diff
@@ -1,10 +1,84 @@
 import React from 'react';
 import './App.css';
+import Button from "./Button";
+import TheHandyBag from './assets/bag_1.png';
+import TheStylishBag from './assets/bag_2.png';
+import TheTrendyBag from './assets/bag_3.png';
+import TheSimpleBag from './assets/bag_4.png';
+import BrandImg from './assets/brand.png';
+import StoryImg from './assets/our_story.png';
+import Product from "./Product";
+import Tile from "./Tile";
+
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button buttonName="to the collection" disabled={false}/>
```

Comment:
In de uitwerkingen is 'isDisabled' gebruikt. waarschijnlijk verschil html/js. Zou niet weten wat beter is?